spec/features/visitor_can_browse_items_by_categories_spec.rb

```diff
@@ -3,7 +3,11 @@
 RSpec.describe "VisitorCanBrowseItemsByCategories", type: :feature do
   feature "visitor" do
     let!(:category) { Category.create(name: 'Lenses') }
-    let!(:item_one) { Item.create(name: '70-200 Canon L', description: "portrait lens", price: 12, category_id: category.id) }
+    let!(:item_one) { Item.create(name: '70-200 Canon L',
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.